libs/domain/cart/list/list.component.ts

```diff
@@ -0,0 +1,32 @@
+import { CartService } from '@spryker-oryx/cart';
+import { resolve } from '@spryker-oryx/di';
+import { ContentMixin, LayoutMixin } from '@spryker-oryx/experience';
+import { signal } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+export class CartListComponent extends LayoutMixin(ContentMixin(LitElement)) {
+  protected cartService = resolve(CartService);
+  protected $carts = signal(this.cartService.getCarts());
+
+  protected override render(): TemplateResult | void {
+    const carts = this.$carts();
+
+    // return html`
+
+    // `;
+
+    return this.renderLayout({
+      template: html`
+        ${repeat(
+          carts ?? [],
+          ({ id }) => id,
+          ({ id }) =>
+            html`<oryx-cart-list-item cartId=${id}></oryx-cart-list-item>`
+        )}
+        ${unsafeHTML(`<style>${this.layoutStyles()}</style>`)}
```

Comment:
Styles are rendered as part of renderLayout. I see the confusion, you can deprecate `layoutStyles` in the mixin interface if you like, so we won't do this again. Will talk to Alexey about it though.

Reply:
Styles was dropped. I checked other components, and some of them are using the same pattern with layoutStyles. If there is something obsolete, better to deprecate it and make some cleanup in the separate ticket, because I'm not sure when this story will be ready to be merged to dev, there ads too many missed things